Confus/ConfusServer/Networking/Connection.cpp

```diff
@@ -9,7 +9,7 @@ namespace ConfusServer
 {
     namespace Networking
     {
-        Connection::Connection()
+        Connection::Connection(Player* a_MainPlayer, Player* a_SecondPlayer)
```

Comment:
That's quite the extreme limitation to two players in the entire game. You really want to have some ID for the particular client that sent the attack and match this to the correct Player instance on the server.